### PR TITLE
Sprint5 81 conditional display bottom nav

### LINKED ISSUE #81: The wvr-header component's bottom-nav section should display conditionally
Current the wvr-header's bottom-nav appears regardless of the presence of absence of transcluded content within that section.

The wvr-header should expose a boolean input to suppress the display of the bottom-nav.

Additionally a boolean input such as `suppressBottomNavIfEmpty` could be expose to allow the conditional display of the bottom nav based on the presence or absence of transcluded content within the bottom-nav.  This second option is prefered.

Default behavior: only display if content exists.
Boolean displayBottomNav: if true allways display, if false never display, if undefined conditionally display.

### PR DESCRIPTION
**Pull request by rmathew1011**
Resolves #81

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -13,7 +13,7 @@
 </head>
 
 <body>
-  <wvr-header>
+  <wvr-header display-bottom-nav="false">
```
Comment (jeremythuff) [inline src/index.html:16]: This is not needed

Comment (rmathew1011) [reply src/index.html]: Removed this attribute

File: projects/wvr-elements/src/lib/wvr-header/wvr-header.component.ts [hunk 2]
```diff
@@ -56,11 +56,19 @@ export class WvrHeaderComponent {
...
+  @Input() displayBottomNav: 'true' | 'false';
+
   /**
    * The weaver header component constructor
    * @param domSanitizer: DomSanitizer - this parameter is injected to the weaver component instance.
    */
-  constructor(private readonly domSanitizer: DomSanitizer) {
+  constructor(private readonly domSanitizer: DomSanitizer, private readonly elementRef: ElementRef) {
+  }
+
```
Comment (jeremythuff) [inline projects/wvr-elements/src/lib/wvr-header/wvr-header.component.ts:67]: We should have a comment here

File: projects/wvr-elements/src/lib/wvr-header/wvr-header.component.ts [hunk 3]
```diff
@@ -56,11 +56,19 @@ export class WvrHeaderComponent {
   /** Allows for the override of the --bottom-nav-padding css variable. Default:  --wvr-navbar-padding */
   @HostBinding('style.--bottom-nav-padding') @Input() bottomNavPadding;
```
Comment (jeremythuff) [inline projects/wvr-elements/src/lib/wvr-header/wvr-header.component.ts:58]: A comment is needed here as well

**Pull request merged**